lib/krane/deploy_task.rb

```diff
@@ -70,7 +70,19 @@ def predeploy_sequence
       ).map { |r| [r, default_group] }
 
       after_crs = %w(
+        Deployment
+        Service
+        Ingress
         Pod
+        Job
+        CronJob
+        DaemonSet
+        HorizontalPodAutoscaler
+        PodDisruptionBudget
+        PodSetBase
+        PodTemplate
+        ReplicaSet
+        StatefulSet
       ).map { |r| [r, default_group] }
```

Comment:
```suggestion
      ).select(&:predeployed?).map { |r| [r, default_group] }
```

Reply:
Changing this functionality goes a bit deeper because we don't have access to these resources at this step in the deployment, so we're going to scrap this change for now and propose it to the Krane maintainers in the PR.